packages/lesswrong/lib/sql/LogTableQuery.ts

```diff
@@ -0,0 +1,19 @@
+import Query from "./Query";
+import Table from "./Table";
+
+/**
+ * Builds a Postgres query to alter a table to be LOGGED (ie to use the Write-Ahead Log).
+ * This is the default, so this query is only useful for reverting a table to the default.
+ */
+
+class LogTableQuery<T extends DbObject> extends Query<T> {
```

Comment:
`LogTableQuery` and `UnlogTableQuery` could probably just be one query with a `logged: boolean` parameter. I don't know. Both ways are probably fine.